pkg/cmd/docgen/funcs.go

```diff
@@ -182,7 +182,7 @@ func generateFunctions(from []string, categorize bool) []byte {
 		}
 		seen[name] = struct{}{}
 		props, fns := builtinsregistry.GetBuiltinProperties(name)
-		if !props.ShouldDocument() {
+		if !props.ShouldDocument() || strings.HasPrefix(name, "crdb_internal.") {
```

Comment:
`strings.HasPrefix(name, "crdb_internal.")` can happen before checking `seen` since it does not depend on any other state.

Could you also add a comment why it is skipped?